megamek/src/megamek/client/ui/swing/MegaMekGUI.java

```diff
@@ -675,71 +676,99 @@ public String getDescription() {
         }
 
         // popup options dialog
-        GameOptionsDialog god = new GameOptionsDialog(frame, g.getOptions(), false);
-        god.update(g.getOptions());
-        god.setEditable(true);
-        god.setVisible(true);
-        for (IBasicOption opt : god.getOptions()) {
-            IOption orig = g.getOptions().getOption(opt.getName());
-            orig.setValue(opt.getValue());
+        if (!sl.hasFixedGameOptions()) {
+            GameOptionsDialog god = new GameOptionsDialog(frame, g.getOptions(), false);
+            god.update(g.getOptions());
+            god.setEditable(true);
```

Comment:
These two are odd... GameOptionsDialog::update nor GameOptionsDialog::setEditable should need to be called in this case.

Reply:
I did not think about these lines one bit, I only placed them in the if {} to make the dialog skippable.